R/derive_vars_computed.R

```diff
@@ -0,0 +1,211 @@
+#' Adds a Variable Computed from the Analysis Value of one or more Parameters
+#'
+#' Adds a variable computed from the analysis value of one or more parameters.
+#' It is expected that the value of the new variable is defined by an expression
+#' using the analysis values of other parameters. For example Body Mass Index at
+#' Baseline (BMIBL) can be derived from of HEIGHT and WEIGHT parameters.
+#'
+#' @param dataset
+#'
+#'   The variables specified by the `by_vars` parameter are expected.
+#'
+#'   The variable specified by `by_vars` must be a unique key of the input
+#'   dataset after restricting it by the filter condition (`filter` parameter).
```

Comment:
Should be removed (there is no `filter` parameter and `by_vars` doesn't need to be a unique key).